zshrc

```diff
@@ -45,12 +45,13 @@ unsetopt nomatch
 
 # vi mode
 bindkey -v
-bindkey "^F" vi-cmd-mode
-bindkey jj vi-cmd-mode
```

Comment:
As the person who added this: noooooooooo!

(But really a thoughtbotter who uses these dotfiles and uses the `jj` binding should weigh in.)